provider/adapters/src/fallback/adapter.rs

```diff
@@ -113,22 +116,29 @@ impl<P> LocaleFallbackProvider<P> {
         }
     }
 
-    fn run_fallback<F, R>(
+    fn run_fallback<F1, F2, R>(
         &self,
         key: ResourceKey,
         base_req: &DataRequest,
-        mut f: F,
+        mut f1: F1,
+        mut f2: F2,
```

Comment:
I have a `DataResponse` in 3/4 cases but an `AnyResponse` in one. I was considering implementing `AsMut<DataResponseMetadata>` but felt it was better to do this little workaround all in private code in this crate rather than adding some weird impl to the main crate

Reply:
Ah right